.github/workflows/deploy_cms.yaml

```
@@ -0,0 +1,90 @@
+name: Deploy CMS
+
+on:
+  push:
+        branches: 'main'
+        # Run only if there are at least one change matching the following paths
+        paths:
+          - 'apps/strapi-cms/**'
+          - '.github/workflows/deploy_cms.yaml'
+
+  workflow_dispatch:
+    inputs:
+      logLevel:
+        description: 'Log level'
+        required: true
+        default: 'warning'
+        type: choice
+        options:
+        - info
+        - warning
+        - debug
```

```suggestion
        options:
        - warning
```
or maybe we can delete `logLevel` from this workflow